rustfmt.toml

```diff
@@ -2,7 +2,6 @@ format_code_in_doc_comments = true
 group_imports = "StdExternalCrate"
 imports_granularity = "module"
 normalize_comments = true
-reorder_impl_items = true
```

Comment:
This option rearrange items in trait `impl` blocks alphabetically, which quite often doesn't make sense (especially since this can be different from the order they are defined in `trait`).